pandas/_typing.py

```diff
@@ -297,6 +297,9 @@ def closed(self) -> bool:
 else:
     TakeIndexer = Any
 
+# Shared by functions such as drop and astype
+IgnoreRaise = Literal["ignore", "raise"]
```

Comment:
I also added it to a few other functions (there might be more that only accept ignore/raise).